alonzo/formal-spec/utxo.tex

```diff
@@ -231,6 +254,12 @@ \subsection{Plutus Script Validation}
   the consensus's forecast window)
 \end{itemize}
 
+Note that because of this potential slot length change nuance, Plutus scripts will
```

Comment:
Is this actually explained somewhere in this spec already? If not, maybe a short explanation of the issue would be nice.

Reply:
the paragraphs above explain why we translate the validity interval to time. This sentence is to highlight that we dont pass anything at all that mentions slots